.github/workflows/draft-release.yml

```diff
@@ -24,18 +24,18 @@ jobs:
       # write permission is required for autolabeler
       # otherwise, read permission is required at least
       pull-requests: write
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-22.04
```

Comment:
Is there an issue if we use latest! or do we want to force to use ubuntu 22.04 for the draft release?

Reply:
We're using ubuntu 22.04 for all the rest of the actions.